python_modules/libraries/dagster-components/dagster_components/cli/check.py

```diff
@@ -71,6 +72,22 @@ def format_indented_error_msg(col: int, msg: str) -> str:
 OFFSET_LINES_AFTER = 3
 
 
+def stacktrace_to_formatted_error(path: str, exception_msg: str, trace: list[str]) -> str:
```

Comment:
cc: @gibsondan do we have a standardized way of doing this? Similar functionality to your demo

Reply:
there is some imperfect code that tries to do something similar in the internal repo here: https://github.com/dagster-io/internal/blob/3a84faee2a3708b820f0273e5757a059ae3d454e/dagster-cloud/python_modules/dagster-cloud/dagster_cloud/util/errors.py#L78-L124